R/theme.r

```diff
@@ -148,6 +148,9 @@
 #' @param strip.placement placement of strip with respect to axes,
 #'    either "inside" or "outside". Only important when axes and strips are
 #'    on the same side of the plot.
+#' @param strip.clip should strip background edges and strip labels be clipped
+#'   to the extend of the strip background? Options are `"on"`, `"off"` and
+#'   `"inherit"`.
```

Comment:
What does inherit mean?

Reply:
It means the same thing as for the `clip` argument in `grid::viewport()`: it inherits the clipping from the parent viewport. It was suggested to allow this in [this comment](https://github.com/tidyverse/ggplot2/pull/4125#issuecomment-656294744).

Reply:
Can you explain that in the docs?